$NetBSD: patch-florist.gpr,v 1.1 2014/04/30 16:27:04 marino Exp $

  Set the GPR file with standard locations for Ada projects

--- florist.gpr.orig	2012-05-10 13:32:15.000000000 +0000
+++ florist.gpr
@@ -1,12 +1,9 @@
 project Florist is
 
-   for Source_Dirs use ("../../floristlib");
-   for Object_Dir use "../../floristlib";
-
-   for Externally_Built use "True";
-
-   package Linker is
-     for Linker_Options use ("-lflorist");
-   end Linker;
+   for Languages use ("ada");
+   for Source_Dirs use ("../../include/florist");
+   for Library_Name use "florist";
+   for Library_Dir use  "../../lib/florist";
+   for Externally_Built use "true";
 
 end Florist;
